src/Dfe.PlanTech.Infrastructure.Contentful/Content/Renderers/Models/PartRenderers/ParagraphRenderer.cs

```diff
@@ -24,7 +24,7 @@ public override StringBuilder AddHtml(RichTextContent content, IRichTextContentP
         {
             stringBuilder.Append("<p");
             _options.AddClasses(stringBuilder);
-            stringBuilder.Append("\">");
+            stringBuilder.Append('>');
```

Comment:
The original version was like that because of `RichTextPartRendererOptions`. Might need to append the `"` there, and ensure that any other `<WHATEVER>Renderer` of it don't require updating as well

Reply:
Yeah there were only two uses of it and I updated both to not add the `"` and have the `AddClasses` method to it instead

Reply:
Missed that for some reason. good job